### PR TITLE
(maint) Add Internalizer Vagrant environment

### PR DESCRIPTION
**Pull request by pauby**
Added new internalizer Vagrant environment.

### CODE REVIEW

File: SetupResources.ps1 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+@( 'chocolateygui', 'dotnet4.5.2', 'chocolatey.server', 'dotnet4.6.1', 'KB2919355', 'KB2919442',
+    'baretail', 'dotnetversiondetector', 'notepadplusplus', 'vscode', 'git', 'launchy', 'bginfo' ) | ForEach-Object {
+    if ($Force.IsPresent -or (-not (Test-Path -Path (Join-Path -Path $packagesPath -ChildPath "$_.*.nupkg")))) {
+        choco download $_ --internalize --internalize-all-urls --append-use-original-location --output-directory=$packagesPath --source="'https://chocolatey.org/api/v2/; https://licensedpackages.chocolatey.org/api/v2/''"
+    }
+}
+
+# Download .NET 4 Installer
+$NetFx4Url = 'http://download.microsoft.com/download/9/5/A/95A9616B-7A37-4AF6-BC36-D6EA96C8DAAE/dotNetFx40_Full_x86_x64.exe'
```
Comment (gep13) [inline SetupResources.ps1]: Why is this required?

File: scripts/InstallNet4.ps1 [hunk 2]
```diff
@@ -0,0 +1,52 @@
+$NetFx4ClientUrl = 'http://download.microsoft.com/download/5/6/2/562A10F9-C9F4-4313-A044-9C94E0A8FAC8/dotNetFx40_Client_x86_x64.exe'
```
Comment (gep13) [inline scripts/InstallNet4.ps1:1]: Similar question to above.... where did this requirement come from?

File: scripts/InstallBGInfo.ps1 [hunk 3]
```diff
@@ -0,0 +1,9 @@
+if ($null -eq (Get-Command -Name 'choco.exe' -ErrorAction SilentlyContinue)) {
+    Write-Warning "Chocolatey not installed. Cannot install BGInfo."
+}
+else {
+    choco upgrade bginfo -y --source="'c:\packages'"
```
Comment (gep13) [inline scripts/InstallBGInfo.ps1]: @pauby I think this shouldn't be pointed directly at c:\packages

**Review by gep13 — CHANGES_REQUESTED** [review]

File: scripts/InstallNet4.ps1 (hunk 2 above)
Comment (pauby) [reply scripts/InstallNet4.ps1:1]: Resolved via Zoom.

File: scripts/InstallBGInfo.ps1 (hunk 3 above)
Comment (pauby) [reply scripts/InstallBGInfo.ps1]: @gep13 I've removed that.

**Review by gep13 — APPROVED** [review]

**Comment by gep13** [comment]
@pauby your changes have been merged, thanks for your contribution 👍
